cli/src/commands/parse.rs

```diff
@@ -36,19 +36,19 @@ pub fn handle_parse(matches: &clap::ArgMatches) -> CommandResult {
     Ok(ExitCode::Ok.into())
 }
 /// Attempt to get packages from an unknown lockfile type
-pub fn try_get_packages(path: &Path) -> Result<(Vec<PackageDescriptor>, PackageType)> {
-    print_user_warning!(
-        "Attempting to obtain packages from unrecognized lockfile type: {}",
-        path.to_string_lossy()
-    );
-
```

Comment:
This warning was misplaced here because this code gets run in the `--lockfile-type auto`. In that case, we don't even attempt to check if it's a recognized lockfile and we are using behavior explicitly requested by the user, so a warning is unnecessary.